### PR TITLE
Create issue and PR templates

### PR DESCRIPTION
**Pull request by GormFrank**
Created issue templates for:

- Bug requests;
- Feature requests;
- Other (questions/comments).

See those templates in action here: https://github.com/GormFrank/wet-boew/issues

... and created a default bilingual PR template.

- [x] Look into having two PR templates, one in EN and one in FR

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+---
+
+_(Feel free to write you report in French if you prefer.)_
+
+## Required information
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+If you fond another bug report related to this issue on GitHub, please mention its number e.g. #7890
```
Comment (RobJohnston) [inline .github/ISSUE_TEMPLATE/bug.md:16]: Change "fond" to "find".

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+---
+
+_(Feel free to write your suggestion in French if you prefer.)_
+
+## Required information
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+If you fond another feature request or bug report related to this issue on GitHub, please mention its number e.g. #7890
```
Comment (RobJohnston) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]: Change "fond" to "find".

**Review by RobJohnston — COMMENTED** [review]
Consider adding a template for a pull request too.  Something like the following "pull_request_template.md ":

Summary of the changes (Less than 80 chars)
 - Detail 1
 - Detail 2

Addresses #bugnumber (in this specific format)

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (RobJohnston) [reply .github/ISSUE_TEMPLATE/bug.md:16]: On second thought, was it supposed to be "find" or "found"?

Comment (GormFrank) [reply .github/ISSUE_TEMPLATE/bug.md:16]: found. Thanks!

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+name: Bug
```
Comment (duboisp) [inline .github/ISSUE_TEMPLATE/bug.md:2]: Can we put bilingual name like "Bug / Problème"?

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by delisma** [comment]
Le nom du réertoire devrait être au pluriel `ISSUE_TEMPLATES`

**Review by delisma — CHANGES_REQUESTED** [review]
~~Le nom du répertoire devrait être au pluriel `ISSUE_TEMPLATES`~~

**Review by duboisp — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-fr.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bogue
+about: Générer un rapport de bogue pour nous aider à nous améliorer
+title: "[BUG] Nom de la fonctionnalité - Courte description du bogue"
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/bug-fr.md:4]:
```suggestion
title: "[BOGUE] Nom de la fonctionnalité - Courte description du bogue"
```

File: .github/ISSUE_TEMPLATE/feature-request-fr.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+name: Demande de fonctionnalité
+about: Suggérer une idée pour le projet
+title: "[FEATURE] Courte description de la fonctionnalité"
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/feature-request-fr.md:4]:
```suggestion
title: "[FONCTION] Courte description de la fonctionnalité"
```

File: .github/ISSUE_TEMPLATE/other-fr.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+name: Autre
+about: Poser une question, laisser un commentaire, demander de l'aide
+title: "[OTHER] Résumé de la requête"
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/other-fr.md:4]:
```suggestion
title: "[AUTRE] Résumé de la requête"
```

File: .github/ISSUE_TEMPLATE/bug-en.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+## Required information
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+If you found another bug report related to this issue on GitHub, please mention its number e.g. #7890
+
+**To Reproduce**
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/bug-en.md:16]:
```suggestion
**To Reproduce**
<!-- Mandatory -->
```

File: .github/ISSUE_TEMPLATE/bug-en.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+about: Create a bug report to help us improve
+title: "[BUG] Feature name - Short description of the bug"
+labels: ''
+assignees: ''
+
+---
+
+## Required information
+
+**Describe the bug**
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/bug-en.md:12]:
```suggestion
**Describe the bug**
<!-- Mandatory -->
```

File: .github/ISSUE_TEMPLATE/bug-fr.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+---
+
+## Information nécessaire
+
+**Description du bogue**
+Une description claire et concise du bogue.
+Si vous avez trouvé un autre rapport de bogue relié à ceci sur GitHub, veuillez mentionner son numéro ex. : #7890
+
+**Pour le reproduire**
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/bug-fr.md:16]:
```suggestion
**Pour le reproduire**
<!-- Obligatoire -->
```

File: .github/ISSUE_TEMPLATE/bug-fr.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+about: Générer un rapport de bogue pour nous aider à nous améliorer
+title: "[BUG] Nom de la fonctionnalité - Courte description du bogue"
+labels: ''
+assignees: ''
+
+---
+
+## Information nécessaire
+
+**Description du bogue**
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/bug-fr.md:12]:
```suggestion
**Description du bogue**
<!-- Obligatoire -->
```

File: .github/ISSUE_TEMPLATE/feature-request-en.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+about: Suggest an idea for this project
+title: "[FEATURE] Short description of the feature"
+labels: ''
+assignees: ''
+
+---
+
+## Required information
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/feature-request-en.md:12]:
```suggestion
**Is your feature request related to a problem? Please describe.**
<!-- Mandatory -->
```

File: .github/ISSUE_TEMPLATE/feature-request-fr.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+labels: ''
+assignees: ''
+
+---
+
+## Information nécessaire
+
+**Est-ce que votre demande de fonctionnalité est reliée à un problème ? Veuillez élaborer.**
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/feature-request-fr.md:12]:
```suggestion
**Est-ce que votre demande de fonctionnalité est reliée à un problème ? Veuillez élaborer.**
<!-- Mandatory -->
```

File: .github/ISSUE_TEMPLATE/other-en.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
+---
+name: Other
+about: Ask a question, leave a comment, seek guidance
+title: "[OTHER] Summary of the purpose"
+labels: ''
+assignees: ''
+
+---
+
+## Describe the purpose of this issue
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/other-en.md:10]:
```suggestion
## Describe the purpose of this issue
<!-- Mandatory -->
```

File: .github/ISSUE_TEMPLATE/other-fr.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+labels: ''
+assignees: ''
+
+---
+
+## Objectif de cette requête
```
Comment (delisma) [inline .github/ISSUE_TEMPLATE/other-fr.md:10]:
```suggestion
## Objectif de cette requête
<!-- Obligatoire -->
```

**Review by duboisp — APPROVED** [review]

**Comment by GormFrank** [comment]
Approved during technical review. @duboisp Please merge.

File: .github/ISSUE_TEMPLATE/bug-en.md (hunk 8 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/bug-en.md:12]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/bug-en.md (hunk 7 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/bug-en.md:16]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/bug-fr.md (hunk 10 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/bug-fr.md:12]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/bug-fr.md (hunk 9 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/bug-fr.md:16]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/feature-request-en.md (hunk 11 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/feature-request-en.md:12]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/feature-request-fr.md (hunk 12 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/feature-request-fr.md:12]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/other-en.md (hunk 13 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/other-en.md:10]: Having it in the preceding section header is sufficient.

File: .github/ISSUE_TEMPLATE/other-fr.md (hunk 14 above)
Comment (duboisp) [reply .github/ISSUE_TEMPLATE/other-fr.md:10]: Having it in the preceding section header is sufficient.

**Review by duboisp — APPROVED** [review]

**Pull request merged**